src/components/common/form/OtpInput.js

```diff
@@ -109,19 +125,30 @@ const SingleOtpInput = withStyles(getSingleOtpInputStylesFromProps)((props: Sing
   if (hasErrored && errorStyle) {
     inputStyles.push(errorStyle)
   }
+  const inputProps = {
+    style: inputStyles,
+    maxLength: '1',
+    disabled: isDisabled,
+    value: value || '',
+    returnKeyType: 'next',
+    placeholder,
+    focus,
+  }
+  const inputValidations = isInputNum
+    ? {
+        min: 0,
+        max: 9,
+        pattern: /\d/g,
+        type: 'number',
+        keyboardType: 'phone-pad',
+      }
+    : {
```

Comment:
What's the usecase for `isInputNum=== false`? Are we using this in any place?

Reply:
not, but I tried to make the component as reusable as possible, so now we send the code as numbers, but if in the future we want to send alphabetic codes, we can do that easily...